### PR TITLE
[RW-780][risk=no] Document common-api DB access

### PR DESCRIPTION
**Pull request by calbach**
This reflects my current understanding from talking to Dan. Please let me know if there's more detail I can add or if anything here sounds incorrect.

### CODE REVIEW

**Comment by calbach** [comment]
@biopete or @SrushtiGangireddy please take a look

**Comment by SrushtiGangireddy** [comment]
Looks good to me.

**Comment by calbach** [comment]
@SrushtiGangireddy usually I wait for an explicit approval via the GitHub UI before submitting. I'm not sure whether the lack of approval intentional (indicating I should wait for Peter's review). If this is good to merge, could you please approve?

File: common-api/README.md [hunk 1]
```diff
@@ -0,0 +1,10 @@
+# Common API
+
+Common utilities, services, data models between the Workbench and Public Data Browser.
+
+## Data model distinction
+
+Some models, e.g. cdr/model/Concept.java, may be used with either a registered CDR
+database or a public CDR database (depending on which app it is being used in). For the
+public DB, the data may have gone through additional deidentification, e.g. for Concept the
```
Comment (biopete) [inline common-api/README.md]: "may have gone through"  should be "has gone through" .. :) There is no may have.

**Comment by biopete** [comment]
Sorry CH. didn’t notice i was on review. I approved with that one comment. Feel free to slack me if you add me to review too. Between github and my emails floods I don’t always see notifications i’m requested. 

Thanks

File: common-api/README.md (hunk 1 above)
Comment (calbach) [reply common-api/README.md]: I meant this on a table-to-table basis, e.g. the concept relationship table wouldn't have gone through any deidentification. Tweaked wording.

**Pull request merged**